### DISCUSSION TITLE
Welcome to TOSCA-2 Discussions!

### DISCUSSION THREAD

**Discussion by orttak** [digitalcityscience/TOSCA-2#118] [Announcements]
Here's a GitHub discussion announcement text for introducing **Tosca**, the open-source Web GIS application, to your end users:

---

# 🎉 Introducing Tosca: Your New Open-Source Web GIS Solution 🌍

Hello everyone! 👋

We’re excited to announce **Tosca**, an open-source Web GIS application designed to streamline your spatial data workflows and empower you with a robust, user-friendly mapping platform.

**Comment by ArjamaM** [comment]
Hello, and thank you for being part of TOSCA's open-source community! If you are on Github, please contribute your ideas, feedback, feature requests around TOSCA here!
